### DISCUSSION TITLE
Advice for upgrade

### DISCUSSION THREAD

**Discussion by drugsnhugs** [Chia-Network/chia-blockchain#3663] [Q&A]
Hello,

I got a Ryzen 5600X on an ASUS TUF GAMING B550-PLUS mobo and some 8gb of 2300 mhz ram.

Currently plotting on an 480 Gb SATA ssd and getting around 1 plot every 5hrs.

I was thinking of upgrading by buying:

SSD ADATA XPG Gammix S11 Pro 1TB PCI Express 3.0 x4 M.2 2280

Ram Corsair Vengeance LPX Black 16GB DDR4 3600MHz CL20 Dual Channel Kit

With the 2 ssd's and that ram i will be able to plot 4 in parallel right?

Thanks!

**Comment by Muffexx** [comment]
Hello @drugsnhugs,

16GB for four plots is very tight, since the operating system also has to have access to resources. That's why I would go with more RAM.
The rest of the components look okay so far.

regards
Muffexx

**Reply by drugsnhugs** [reply]
i understand. 
Very tight budget here so what if i used the 8 gb i already heave together with the new 16 gb? 
Would that affect the frequency of the more performant memories?

**Reply by Muffexx** [reply]
It depends on the frequency and timings of both kits.
The motherboard will probably take over the speed of the slower kit for both. This should only marginally affect the plotting performance.
I would try to use both kits if I were you. This way you could easily plot on four queues.

**Reply by drugsnhugs** [reply]
Well then, i will buy 16 Gb of 2133 mhz. 
That way i would cut some of the expenses. 
I was going for 3600 mhz because i heard on some yt video that ryzen plots better with bigger frequency. 

Thank you!
